examples/tut_add-vectors.cpp

```diff
@@ -51,6 +55,11 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 
   std::cout << "\n\nRAJA vector addition example...\n";
 
+#if defined(RAJA_ENABLE_SYCL)
+  memoryManager::sycl_res = new camp::resources::Resource{camp::resources::Sycl()};
```

Comment:
What does this do?

Reply:
The camp resources do memory alloc/dalloc/copy, etc. We use those in the parameterized RAJA tests.